client/src/Service/Csv/OldAdminUserCsvGenerator.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
```

Comment:
Could we add:

```suggestion
declare(strict_types=1);

```

This will ensure we're type safe when using this class.